app/src/main/java/openfoodfacts/github/scrachx/openfood/views/HistoryScanActivity.java

```diff
@@ -3,6 +3,8 @@
 
 import android.Manifest;
 import android.app.Activity;
+import android.app.NotificationManager;
```

Comment:
The app crashes trying to export the history (emulator, Android 4.4.2, Galaxy Nexus API 19) - can you check it please?

```
02-20 23:56:09.134 4733-4733/openfoodfacts.github.scrachx.openfood E/dalvikvm: Could not find class 'android.app.Notification$Action$Builder', referenced from method android.support.v4.app.NotificationCompatBuilder.addAction
02-20 23:56:09.174 4733-4733/openfoodfacts.github.scrachx.openfood E/AndroidRuntime: FATAL EXCEPTION: main
                                                                                     Process: openfoodfacts.github.scrachx.openfood, PID: 4733
                                                                                     android.app.RemoteServiceException: Bad notification posted from package openfoodfacts.github.scrachx.openfood: Couldn't expand RemoteViews for: StatusBarNotification(pkg=openfoodfacts.github.scrachx.openfood user=UserHandle{0} id=0 tag=null score=0: Notification(pri=0 contentView=openfoodfacts.github.scrachx.openfood/0x1090064 vibrate=null sound=null defaults=0x0 flags=0x0 kind=[null] 1 action))
                                                                                         at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1363)
                                                                                         at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                                         at android.os.Looper.loop(Looper.java:136)
                                                                                         at android.app.ActivityThread.main(ActivityThread.java:5017)
                                                                                         at java.lang.reflect.Method.invokeNative(Native Method)
                                                                                         at java.lang.reflect.Method.invoke(Method.java:515)
                                                                                         at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:779)
                                                                                         at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:595)
                                                                                         at dalvik.system.NativeStart.main(Native Method)x§
```

Reply:
ohhh... i figured it out, that is because we are using vector drawable of share image, and vector drawable are introduced, i know we are using backward support library for VectorDrawables, but i tried it without vector drawable it was working fine in 4.4(Kitkat) or also with normal PNGs.